Console/Helpers/ViewModels/Connection.cs

```diff
@@ -42,7 +44,7 @@ public Connection(IPHelper.I_OWNER_MODULE ownerMod)
                 Path = "Unresolved"; //FIXME: Use something else?
             }
 
-            if (ownerMod.OwnerModule == null)
+            if (ownerMod.OwnerModule is null)
             {

```

Comment:
Is there an advantage using "is null" and is there also a "isNot"? Not sure I like it 🤔

Reply:
"is null" is the "new" way to check for null as it offers better performance (uses what's called pattern matching instead of regular Equals overloads).
I wasn't sure I liked it either at the beginning, now I do ;)

"is not null" exists (sort of), and is in fact the _is object_ pattern matching test. I don't like this one for sure as it's not really readable, but according to many sources (including .Net engineers), it would be the way to go... this is why I didn't replace _!= null_ by _is object_ yet.